### ISSUE TITLE
A problem of Supervoxel generation

### ISSUE THREAD

**Issue by lijing-coder** [sha168/ADNet#3]
Supervoxels were generated for 3D data in AD-Net, but when I displayed the images in the generated Supervoxels, the quality of the generated Supervoxels was not good in terms of pure visual perception. I also displayed the superpix of ouyang under the same image, which felt good. Specific visible images.

![image](https://user-images.githubusercontent.com/66677296/234285345-558735f1-28de-41e1-89d1-961873437380.png)
![image](https://user-images.githubusercontent.com/66677296/234285480-0f8cf277-66b9-41e7-a281-fd2d3a520337.png)
![a982f95ec063a13b31d8c9c7b07c9dac](https://user-images.githubusercontent.com/66677296/234296943-8c9c724b-4161-49b0-a5cf-f184eb38959a.png)

This picture is the 33rd image of image_37.nii.gz, the top two images are taken from the supervoxel generated by AD-Net, and the following images are the superpix generated by ouyang.

This picture is the ground truth.
![image](https://user-images.githubusercontent.com/66677296/234287882-3a667652-33af-4fb8-bb65-59a0847ce833.png)

**Comment by lijing-coder** [comment]
Have you ever compared using the superpix generated by ouyang directly for AD-Net training?

**Comment by lijing-coder** [comment]
Thank you for your excellent work and look forward to your answer!

**Comment by shenqq377** [comment]
> Have you ever compared using the superpix generated by ouyang directly for AD-Net training?

Yeah, We have found the similar problem in some hard cases when you process 3D data. We didn't compare these two approaches. For a certain organ, we get different superpixel class ids on different slices by using superpixel segmentation slice by slice, thus we are unable to select adjacent 3 slices based on a specific superpixel classes id. It is a good idea to do the comparison.
